ibis/util.py

```diff
@@ -73,6 +74,36 @@ def is_one_of(values: Sequence[T], t: Type[U]) -> Iterator[bool]:
 all_of = toolz.compose(all, is_one_of)
 
 
+def coerce_to_dataframe(data: Any, names: List[str]) -> pd.DataFrame:
```

Comment:
since this is *only* used in pandas backend can you put it right about coerce_to_output, or even better integrate so you don't need this (e.g. *just* use coerce_to_output), but this could be a helper. is this possible?

Reply:
Actually, this is also used by the pyspark backend to handle struct output from UDFs, so I left it in top level (ibis.util). Do you think we should still move it to pandas backend namespace?